### PR TITLE
[sailfish-browser] Keep the device awake during WebRTC calls. JB#56332

### PR DESCRIPTION
**Pull request by d-grigorev**

### CODE REVIEW

File: apps/shared/ResourceController.qml [hunk 1]
```diff
@@ -32,10 +32,12 @@ Item {
...
     property bool _isAudioStream
     property bool _isVideoStream
+    property bool _webrtcAudioActive
+    property bool _webrtcVideoActive
 
     function calculateStatus() {
-        var video = false
-        var audio = false
+        var video = _webrtcVideoActive
+        var audio = _webrtcAudioActive;
```
Comment (okodron) [inline apps/shared/ResourceController.qml:40]:
```suggestion
        var audio = _webrtcAudioActive
```

Comment (d-grigorev) [reply apps/shared/ResourceController.qml]: Fixed, thanks.

**Review by okodron — APPROVED** [review]

**Comment by okodron** [comment]
This PR should be duplicated to the `next` branch as well.
Depends on https://github.com/sailfishos/embedlite-components/pull/73

**Comment by d-grigorev** [comment]
I rephrased the commit message and created a copy of this PR for the `next` branch: https://github.com/sailfishos/sailfish-browser/pull/943.

**Review by rainemak — APPROVED** [review]

**Pull request merged**